lib/exchange.js

```diff
@@ -34,9 +34,16 @@ export async function generateResponse({
   return {verifiablePresentationRequest};
 }
 
-// eslint-disable-next-line no-unused-vars
+/**
+ * If an exchangeId is defined in the config return true.
+ *
+ * @param {object} options - Options to use.
+ * @param {string} options.exchangeId - The id of an exchange.
+ *
+ * @returns {boolean} Was the exchangeId in exchanges?
+ */
 export async function exists({exchangeId}) {
-  return true;
+  return (exchangeId in (bedrock.config['vc-exchanger'].exchanges || {}));
```

Comment:
Is this better?

```suggestion
  return !!bedrock.config['vc-exchanger'].exchanges?.[exchangeId];
```

Reply:
I actually think the use of `in` here is better than `!!` mostly due to readability. If javascript has a function that returns true if a property is defined in an object why not use it? This is like using binary not `~` with `indexOf` to get a Boolean when includes would work just fine.